projects/plugins/protect/src/class-jetpack-protect.php

```diff
@@ -305,6 +324,37 @@ public function protect_filter_available_modules( $modules ) {
 		return array_merge( array( self::JETPACK_WAF_MODULE_SLUG ), $modules );
 	}
 
+	/**
+	 * Check for user licenses.
+	 *
+	 * @param  boolean $has_license Check if user has a license.
+	 * @param  object  $licenses List of licenses.
+	 * @param string  $plugin_slug The plugin that initiated the flow.
+	 *
+	 * @return boolean
+	 */
+	public static function jetpack_check_user_licenses( $has_license, $licenses, $plugin_slug ) {
+		if ( Plan::has_required_plan() ) {
+			wp_safe_redirect( '/wp-admin/admin.php?page=jetpack-protect' );
```

Comment:
```suggestion
			wp_safe_redirect( '/wp-admin/admin.php?page=jetpack-protect' );
```

I'm not sure we should be redirecting/exiting inside of the filter here, especially because we're doing a plan check here inside of the license check filter.

It might make sense to keep the existing `siteProductAvailabilityHandler` to check if the site has a plan, and then introduce this filter to check if the site has an available license. What do you think?